### ISSUE TITLE
Highlighting with \h in Linewise visual mode generates error on attempt to restore visual mode

### ISSUE THREAD

**Issue by bpstahlman** [bpstahlman/txtfmt#14]
**Background:** Found this issue while attempting to reproduce an issue reported by Ben Basuni. Not sure yet whether Ben's issue has same root cause.

**Version:** Issue exists in Txtfmt version 3.0

**To reproduce:** Use linewise visual mode (V) to select the region for highlighting with \h. Highlighting will succeed, but the following error occurs when Txtfmt attempts to restore visual selection:

```
Error detected while processing function <SNR>50_Highlight_visual:
line   10:
E605: Exception not caught: Highlight_visual: Unable to highlight selection: exception Vim(call):E474: Invalid argument occurred at function <SNR>50_Highlight_visual[8]..<SNR>50_Restore_visual_mode, line 9

```

**Root Cause:** For linewise visual selections, Vim's getpos() function reports "a large number" for the column of mark '>. Vim's help actually does say something about this, but doesn't mention that the number is sufficiently large as to be interpreted as a large _negative_ value, which breaks the logic in Restore_visual_selection.

**Fix:** Since col() is unaffected by this idiosyncrasy, replace the upstream call to getpos() (in Highlight_visual) with separate calls to line() and col(), taking advantage of col()'s ability to accept a list specifying line and symbolic column position.

**Comment by bpstahlman** [comment]
Discovered that the issue also affects smart delete map (\d). In that case, there's no error message, but a spurious line remains after the delete.

**Comment by bpstahlman** [comment]
Fixed in formal release 3.1 (commit 0d996d).
